tensorflow/compiler/xla/service/gpu/tests/gpu_unrolling_test.cc

```diff
@@ -70,32 +70,17 @@ TEST_F(GpuUnrollingTest, UnrollFourTimes) {
   config.set_debug_options(debug_options);
   auto hlo_module = ParseAndReturnVerifiedModule(kAddModule, config).value();
 
-  const std::string pattern1 = R"(
+  CompileAndVerifyIr(std::move(hlo_module),
+                     R"(
 ; CHECK-LABEL: @fusion
 ; CHECK: fadd
 ; CHECK: fadd
 ; CHECK: fadd
 ; CHECK: fadd
 ; CHECK-NOT: fadd
 ; CHECK: }
-<<<<<<< HEAD
-      )";
-
- const std::string pattern2 = R"(
-; CHECK-LABEL: @fusion
-; CHECK: fadd <2 x float>
```

Comment:
Didn't we realize this was needed for MI200 to pass?

Reply:
Not if you check the unoptimized IR.

I submitted the change as we thought in an upstream PR and the reviewer suggested checking the unoptimized IR instead of the optimized IR.

https://github.com/tensorflow/tensorflow/pull/57750